ui/app/models/identity/group.js

```diff
@@ -52,4 +53,34 @@ export default IdentityModel.extend({
   ),
 
   alias: belongsTo('identity/group-alias', { async: false, readOnly: true }),
+  updatePath: queryRecord(
+    'capabilities',
+    context => {
+      const { identityType, id } = context.getProperties('identityType', 'id');
+      //identity/group/id/efb8b562-77fd-335f-a754-740373a778e6
+      return {
+        id: `identity/${identityType}/id/${id}`,
+      };
+    },
+    'id',
+    'identityType'
+  ),
```

Comment:
Thanks for the nudge on this 🙌  - I was able to generalize the pattern we've used across the codebase with a computed macro + the use of a tagged template function to properly interpolate the paths with the named keys. This particular one can even be curried once since the path patterns are the same for all of the identity models ✨. I'll go back in another PR and use it throughout the codebase, but already it's cut down on a lot of code:

https://github.com/hashicorp/vault/pull/4502/commits/feba7a9920092e8fd9190ecb75bd76d2913fe170